lib/segment/src/types.rs

```diff
@@ -983,10 +983,10 @@ impl VectorDataConfig {
 pub enum SparseVectorStorageType {
     /// Storage on disk
     // (rocksdb storage)
-    #[default]
     OnDisk,
     /// Storage in memory maps
     // (blob_store storage)
+    #[default]
     Mmap,
 }
```

Comment:
Note: this also changes the default variant on `SparseVectorStorageType`, I expect no weird side effects because of it.